common/build.gradle

```diff
@@ -25,8 +25,9 @@ dependencies {
   implementation group: 'org.slf4j',                        name: 'slf4j-api',                      version: slf4jApiVersion
   implementation group: 'ch.qos.logback.contrib',           name: 'logback-json-classic',           version: logbackJsonClassicVersion
   implementation group: 'io.micrometer',                    name: 'micrometer-core',                version: micrometerVersion
-  implementation (group: 'io.micrometer',                   name: 'micrometer-jersey2',             version: micrometerVersion) {
-    exclude group: 'org.glassfish.jersey.inject'
+  implementation (group: 'io.micrometer',                   name: 'micrometer-jersey2',             version: micrometerJersey2Version) {
+    exclude group: 'io.micrometer'
```

Comment:
this seems a bit odd. do you remember why was this required?

Reply:
Please see the related wiki page. Quoting here for your convenience:
> Micrometer version inconsistency:
>
>    The latest version of `micrometer-core` is 1.10.4, whereas it is 1.8.13 for `micrometer-jersey2`. It shall be decided if this discrepancy is acceptable, or rather `micrometer-core` can be downgraded to 1.8.13 (if it does not cause any runtime issues, of course).

Reply:
thanks, but I think these stuff should be in the commit message too. it should just be a copy-paste. Although I don't know the answer for the question :)

Reply:
It would be a bit of overkill to copy-paste all of the wiki content into the commit msg.

Reply:
not the whole wiki, just the stuff which is code change related. no one will check the wiki in a few months when trying to figure out why this had been added.